### ISSUE TITLE
SIMPLE Guide To Sending DAO "OUT" Of Poloniex...

### ISSUE THREAD

**Issue by KeyJockey** [ethereum/mist#740]
Hi Folks: Apologies IF this is covered elsewhere but I have srsly tried to look around a lot already, and not found *anything* helpful.

Simple Question (I think) : I just bought some DAO on Poloniex (totally on speculation, I admit, having heard a lot of positive buzz about the whole project) and I just *simply* want to *withdraw* them from Polo, and store them *safely* in my own wallet, or (preferably!) on a "paper wallet" address, as per the usual crypto-currency "best practices" for long-term safe storage of any coin.

So...? What do I need to DO, simple and easy, JUST for *that*... and that alone?  I've downloaded the Mac version of MIST but it's totally confusing and I'm afraid to make a mistake and lose my DAO coins.

Do I really need to run a full blown wallet like Mist for this, with the whole ETH blockchain, etc?

Can anyone make a simple ELI5 guide for this...?

TIA for any/all help (and, I'm sure I'm not the only noob out here looking for this info...!)

**Comment by KeyJockey** [comment]
UPDATE: Okay guys, sorry -- never mind LOL

I found the DAO tab on MyEtherWallet and that worked very quick & simple for what I need.

Mist is cool but totally overkill for just this simple purpose, so thanks anyway but all good now... this issue post can be closed/deleted if you want (or maybe leave it just for info for other noobs like me who may be helped by it...)

**Comment by rokii666** [comment]
Would be great if you post your solution for others. I have same problem - can not find where and how to transfer my DAO from Polo.

**Comment by luclu** [comment]
@rokii666 you can use https://myethereumwallet.com to refund your DAO token as @KeyJockey stated.
But Poloniex won't allow to withdraw just yet: https://twitter.com/Poloniex/status/755807630417862656
